user/pdfgrep/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "pdfgrep"
+pkgver = "2.2.0"
+pkgrel = 0
+build_style = "configure"
```

Comment:
```suggestion
build_style = "gnu_configure"
```

Reply:
Just out of interest: Is there a specific reason, why `gnu_configure` is preferred over just `configure`? It works fine both ways on my end, I'm just curious.

Reply:
Also, configure scripts that are produced using GNU autotools have some common
configuration options. The gnu_configure build style anticipates those, and
sets environment variables and configuration options accordingly.

Case in hand, the `--prefix=/usr` option needs to be specified for almost all
configure scripts generated using autotools, and therefore the gnu_configure
build style sets that automatically, as seen [here](https://github.com/chimera-linux/cports/blob/master/src/cbuild/util/gnu_configure.py#L83-L102)

Reply:
I see, thanks for the explanation!